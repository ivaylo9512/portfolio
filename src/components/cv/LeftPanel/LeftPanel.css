.left-panel{
    width: 69%;
}