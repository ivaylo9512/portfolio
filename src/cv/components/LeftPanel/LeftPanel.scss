.left-panel{
    width: 64%;
    padding-top: 2.1vw;
    
    .github path {
        filter: url(#name-shadow);
    }
}