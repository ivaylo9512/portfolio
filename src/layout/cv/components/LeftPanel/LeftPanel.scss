.left-panel{
    width: 100%;
    
    .github path {
        filter: url(#name-shadow);
    }

    @media only screen and (min-width: 700px) and (orientation: landscape) {
        width: 64%;
        padding-top: 2.1vw;
    }
}