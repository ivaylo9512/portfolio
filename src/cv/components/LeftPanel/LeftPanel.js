import Projects from 'cv/components/Projects/Projects';
import MoreGithub from '../MoreGithub/MoreGithub';
import './LeftPanel.scss';

const LeftPanel = () => {
    return(
        <section className="left-panel">
            <Projects />
            <MoreGithub />
        </section>
    )
}

export default LeftPanel