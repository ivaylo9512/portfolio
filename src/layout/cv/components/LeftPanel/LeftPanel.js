import Projects from '@cv/components/Projects/Projects';
import MoreGithub from '@cv/components/MoreGithub/MoreGithub';
import './LeftPanel.scss';
import { useEffect, useRef } from 'react';
import useResize from 'AppRoot/hooks/useResize';

const LeftPanel = () => {
    const projects = useRef();
    const {dimensions} = useResize();

    useEffect(() => {
        projects.current.style.height = "unset";

        setTimeout(() => {
            projects.current.style.height = projects.current.clientHeight + "px";
            
        }, 1);
    }, [dimensions])

    useEffect(() => {
        projects.current.style.height = projects.current.clientHeight + "px";
    }, [])

    return(
        <section className='left-panel' ref={projects}>
            <Projects />
            <MoreGithub shouldEqual={false} />
        </section>
    )
}

export default LeftPanel