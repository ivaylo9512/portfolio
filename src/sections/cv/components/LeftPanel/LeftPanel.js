import Projects from '@cv/components/Projects/Projects';
import MoreGithub from '@cv/components/MoreGithub/MoreGithub';
import './LeftPanel.scss';

const LeftPanel = () => {
    return(
        <section className='left-panel'>
            <Projects />
            <MoreGithub shouldEqual={false} />
        </section>
    )
}

export default LeftPanel