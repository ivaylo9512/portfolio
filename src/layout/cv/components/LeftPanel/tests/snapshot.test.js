import LeftPanel from '../LeftPanel';
import { shallow } from 'enzyme';

describe('LeftPanel snapshot tests', () => {
    const createWrapper = () => shallow(
        <LeftPanel />
    )

    it('should match snapshot', () => {
        const wrapper = createWrapper();
     
        expect(wrapper).toMatchSnapshot();
    })
});