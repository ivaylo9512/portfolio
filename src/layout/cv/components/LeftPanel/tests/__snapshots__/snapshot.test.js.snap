// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftPanel snapshot tests should match snapshot 1`] = `
<section
  className="left-panel"
>
  <Projects />
  <MoreGithub
    shouldEqual={false}
  />
</section>
`;
